eval/action.yml

```diff
@@ -0,0 +1,66 @@
+name: "Sandard Eval"
+description: "Evaluate Standard based projects"
+
+inputs:
+  key_path:
+    description: "The path to place the Nix secret Key"
+    required: true
+    default: "${{ runner.temp }}/.nix-key"
+  github_pat:
+    description: "GitHub Token for Accessing Private Repos with Nix"
+    required: true
+    default: ${{ github.token }}
+  nix_key:
+    description: "Secret Key used for Nix Package Signing"
+    required: false
+  s3_key:
+    description: "Key to upload to S3 Buckets with Nix"
+    required: false
+  s3_id:
+    description: "ID to upload to S3 Buckets with Nix"
+    required: false
+  cache:
+    description: "Nix cache to optionally upload artifacts to"
+    # auto refers to the local nix store, so nothing is uploaded by default
+    default: auto
+    required: false
+
```

Comment:
I guess we need to define the output (one single big json) here so that we then can take it as input to the subsequent actions?

Reply:
yep, I was wondering why that kept failing :facepalm: